src/CssInliner.php

```diff
@@ -742,7 +742,7 @@ private function sortBySelectorPrecedence(array $first, array $second): int
      */
     private function getCssSelectorPrecedence(string $selector): int
     {
-        $selectorKey = \md5($selector);
+        $selectorKey = $selector;
         if (isset($this->caches[self::CACHE_KEY_SELECTOR][$selectorKey])) {
             return $this->caches[self::CACHE_KEY_SELECTOR][$selectorKey];
```

Comment:
I think we then can directly use `$selector` as array keys and drop the local variable `$selectorKey`.

Reply:
The `$selector` variable is reassigned in the loop in line 756, that's why I kept the `$selectorKey`.